applications/asset_tracker_v2/doc/cloud_wrapper.rst

```diff
@@ -0,0 +1,162 @@
++------------------------------+-----------------------------------------+--------------+-----------+
+
+nRF Cloud topics
+================
+
+For more information regarding topics utilized in the nRF Cloud connection, refer to the following documentation `nRF Cloud MQTT API`_ for details.
+
++------------------------------+----------------------------------------------------+-----------+
+|              Data            |            AWS IoT topic                           | Direction |
++==============================+====================================================+===========+
+| A-GPS requests               | <topic_prefix>/<imei>/d2c                          | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| P-GPS requests               | <topic_prefix>/<imei>/d2c                          | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Neighbor cell measurements   | <topic_prefix>/<imei>/d2c                          | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Button presses               | <topic_prefix>/<imei>/d2c                          | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Sensor/device data           | <topic_prefix>/<imei>/d2c                          | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Device configuration         | $aws/things/<imei>/shadow/update                   | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| Buffered sensor/device data  | <topic_prefix>/<imei>/d2c/batch                    | D2C       |
++------------------------------+----------------------------------------------------+-----------+
+| A-GPS response               | <topic_prefix>/<imei>/c2d                          | C2D       |
++------------------------------+----------------------------------------------------+-----------+
+| P-GPS response               | <topic_prefix>/<imei>/c2d                          | C2D       |
++------------------------------+----------------------------------------------------+-----------+
+| Device configuration updates | $aws/things/<imei>/shadow/delta                    | C2D       |
+|                              +----------------------------------------------------+           |
+|                              | $aws/things/<imei>/shadow/get/accepted             |           |
+|                              +----------------------------------------------------+           |
+|                              | $<imei>/shadow/get/accepted/desired/cfg            |           |
++------------------------------+----------------------------------------------------+-----------+
```

Comment:
**suggestion**: to improve readability, split table in two, one for D2C and one for C2D, remove Direction column.